tests/integrationtest/t/executor/set.test

```diff
@@ -12,26 +12,26 @@ select @@global.tidb_max_delta_schema_count;
 set @@global.tidb_max_delta_schema_count= default;
 
 # TestGetSetNoopVars
-SELECT @@query_cache_type;
-SHOW VARIABLES LIKE 'query_cache_type';
-SET query_cache_type=2;
-SELECT @@query_cache_type;
+SELECT @@innodb_buffer_pool_size;
+SHOW VARIABLES LIKE 'innodb_buffer_pool_size';
+SET GLOBAL innodb_buffer_pool_size=134217728;
+SELECT @@innodb_buffer_pool_size;
```

Comment:
Just curious why adding a test for `innodb_buffer_pool_size` here? :)

Reply:
This is not adding a test, this test is to test get/set of noop vars and it was using `query_cache_type`, which is removed, so changed it so it now does the same test but uses `innodb_buffer_pool_size` instead.